crates/task-impls/src/network.rs

```diff
@@ -38,63 +38,6 @@ use crate::{
     helpers::broadcast_event,
 };
 
-/// quorum filter
-pub fn quorum_filter<TYPES: NodeType>(event: &Arc<HotShotEvent<TYPES>>) -> bool {
-    !matches!(
-        event.as_ref(),
-        HotShotEvent::QuorumProposalSend(_, _)
-            | HotShotEvent::QuorumVoteSend(_)
-            | HotShotEvent::DacSend(_, _)
-            | HotShotEvent::TimeoutVoteSend(_)
-            | HotShotEvent::ViewChange(_)
-    )
-}
-
-/// upgrade filter
-pub fn upgrade_filter<TYPES: NodeType>(event: &Arc<HotShotEvent<TYPES>>) -> bool {
-    !matches!(
-        event.as_ref(),
-        HotShotEvent::UpgradeProposalSend(_, _)
-            | HotShotEvent::UpgradeVoteSend(_)
-            | HotShotEvent::ViewChange(_)
-    )
-}
-
-/// DA filter
-pub fn da_filter<TYPES: NodeType>(event: &Arc<HotShotEvent<TYPES>>) -> bool {
-    !matches!(
-        event.as_ref(),
-        HotShotEvent::DaProposalSend(_, _)
```

Comment:
just to check that I understand correctly, was `DaProposalSend` the only thing that needed to be broadcast to the DA committee (as opposed to broadcasting to the quorum or sent to an individual node directly?)

Reply:
Yeah that's correct